README.rst

```diff
@@ -11,35 +11,35 @@ with interactive Jupyter notebooks in the cloud.
 Index
 -----
 Getting started
- - `Starting guide <s>`_
- - `Units, coordiante system and parameter definitions <s>`_
- - `Handling FITS files <s>`_
+ - `Starting guide <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/GettingStarted/starting_guide.ipynb>`_
+ - `Units, coordiante system and parameter definitions <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/GettingStarted/units_coordinates_parameters_definitions.ipynb>`_
+ - `Handling FITS files <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/GettingStarted/fits_handling_and_extracting_needed_information_from_the_data.ipynb>`_
```

Comment:
here perhaps the text appearing in the link can also be improved

Reply:
How about now?